src/client/datascience/notebook/helpers/helpers.ts

```diff
@@ -71,6 +73,14 @@ const kernelInformationForNotebooks = new WeakMap<
     { metadata?: KernelConnectionMetadata | undefined; kernelInfo?: KernelMessage.IInfoReplyMsg['content'] }
 >();
 
+export function isResourceNativeNotebook(resource: Resource, notebooks: IVSCodeNotebook) {
+    if (!resource) {
+        return false;
+    }
+    return notebooks.notebookDocuments.some(
+        (item) => item.uri.toString().toLowerCase() === resource.toString().toLowerCase()
```

Comment:
I feel like in another code review recently we didn't want to just use toString on URI comparisons. I think we used the arePathsSame api instead.

Reply:
Yes, honestly was being lazy here as we were just trying to identify whehter a file is a notebook or not.
Will fix.